cypress/integration/backoffice/overview.test.spec.js

```diff
@@ -0,0 +1,30 @@
+describe('backoffice overview', () => {
+  const login = () => {
+    cy.visit('/login');
+    cy.get('input#email').type('librarian@test.ch');
+    cy.get('input#password').type('123456');
+    cy.contains('button.primary', 'Sign in').click();
+  };
+
+  it('should be linked from/to the frontsite, can logout', () => {
+    login();
+
+    cy.contains('Hector Nabu').click();
+    cy.contains('Backoffice').click();
+
+    cy.url().should('eq', Cypress.config().baseUrl + '/backoffice');
```

Comment:
nit: we can cache `const baseUrl = Cypress.config().baseUrl`
nit: which one do you find more readable?
```suggestion
    cy.url().should('eq', `${Cypress.config().baseUrl}/backoffice`);
```

Reply:
100% agree! I dislike the `Cypress.config().baseUrl` boilerplate thing, there has to be a better way of doing it.